### ISSUE TITLE
In place change Conv2D

### ISSUE THREAD

**Issue by victorca25** [NVIDIA/partialconv#16]
Hello! First off thanks for your paper and code, it's great insight. 

I wanted to try using PartialConv2D to replace nn.Conv2D for an upscaling task, to test if that would help reduce the artifacts that appear at the image edges (which I guess is caused by the padding), but when I tried to do it, the first loss of the network became extremely large and on the second iteration the loss tensor was just NaN.

Do I have to take anything else into account when testing replacing Conv2D for partial convolution based padding?

Cheers!

**Comment by liuguilin1225** [comment]
Can you make sure the value of self.mask_ratio for the entries in the interior (no need for padding) have value 1?
https://github.com/NVIDIA/partialconv/blob/master/models/partialconv2d.py#L63

If not, you can replace #63 with https://github.com/NVIDIA/partialconv/blob/master/models/partialconv2d.py#L64.

**Comment by victorca25** [comment]
Indeed the values of the self.mask_ratio tensor were all 1 for the entries in the interior, only the edges are 1.5000, while the four corners are 2.2500. But this happened with #63 and also #64, it's having the same behavior. Can I test something else?

**Comment by victorca25** [comment]
I found out what the issue was, I was attempting to resume training on an existing session that was keeping the previous configuration using Conv2D, causing conflicts. Starting from scratch makes it work perfectly.

Will now continue testing to understand better how it works, but the tip was very useful all the same.

Thanks!
